frontend/micro-ui/web/micro-ui-internals/example/public/index.html

```diff
@@ -1,47 +1,47 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-  <meta charset="utf-8" />
-  <link rel="icon" href="https://cdn.jsdelivr.net/npm/@egovernments/digit-ui-css/img/browser-icon.png" />
-  <link
-    href="https://fonts.googleapis.com/css2?family=Roboto+Condensed:wght@400;500;700&family=Roboto:wght@400;500;700&display=swap"
-    rel='stylesheet' type='text/css'>
-  <meta name="viewport" content="width=device-width, initial-scale=1" />
-  <meta name="theme-color" content="#00bcd1" />
-  <title>mSeva</title>
-  <link rel="stylesheet" href="https://unpkg.com/@egovernments/digit-ui-css@1.5.7/dist/index.css" />
-
-  <!-- <link rel="stylesheet" href="https://unpkg.com/@egovernments/digit-ui-css/dist/index.css"/> -->
-  <!-- <script src="https://s3.ap-south-1.amazonaws.com/egov-dev-assets/globalConfigs.js"></script> -->
-  <script src="%REACT_APP_GLOBAL%"></script>
-  <style type="text/css">
-    #logo-img-container {
-      display: none;
-    }
-
-    .selector-button-primary>h2 {
-      margin: 0
-    }
-
-    .menu-wrap>div>p {
-      margin: 0
-    }
-
-    .selector-button-primary-disabled>h2 {
-      margin: 0;
-    }
-
-    .employeeForgotPassword .employee-card-input {
-      margin-bottom: 24px !important;
-      border-left: none;
-    }
-  </style>
-</head>
-
-<body>
-  <noscript>You need to enable JavaScript to run this app.</noscript>
-  <div id="root"></div>
-</body>
-
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="https://karnataka-s3.s3.ap-south-1.amazonaws.com/hrms/favicon.ico" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto+Condensed:wght@400;500;700&family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#00bcd1" />
+    <title>mSeva</title>
+    <link rel="stylesheet" href="https://unpkg.com/@egovernments/digit-ui-css@1.5.7/dist/index.css" />
```

Comment:
check for index.html also

Reply:
i update only the favicon URL the rest thing is same it show green because the formatting.